edb/tools/test/__init__.py

```diff
@@ -27,6 +27,9 @@
 
 from edb.tools import edbcommands
 
+from .decorators import not_implemented  # noqa
+from .decorators import xfail  # noqa
```

Comment:
Alternatively you can define `__all__ = ('not_implemented', 'xfail')` in this file